libres/lib/enkf/block_obs.cpp

```diff
@@ -444,3 +445,20 @@ VOID_MEASURE_UNSAFE(
 VOID_USER_GET_OBS(block_obs)
 VOID_CHI2(block_obs, field)
 VOID_UPDATE_STD_SCALE(block_obs)
+
+class ActiveList;
+namespace {
+void update_std_scaling(py::handle obj, double scaling,
+                        const ActiveList &active_list) {
+    auto *block_obs = reinterpret_cast<block_obs_type *>(
```

Comment:
Could be replaced with `ert::from_cwrap<block_obs_type>(obj)`

Reply:
OK